### ISSUE TITLE
Plugin not working post upgrading to cordova ios 6.x

### ISSUE THREAD

**Issue by ashishkumar2804** [experiencecommerce/cordova-plugin-customconfigparameters#5]
I have upgraded my app to use latest codova ios version. this is because of the deprecated UIwebview API. Post upgrading this plugin is not working and we are getting error for CustomConfigParameters variable not found.

**Comment by DigitalTAC** [comment]
How about window.CustomConfigParameters?

**Comment by ashishkumar2804** [comment]
didn't try that..will try and check if that works

**Comment by ashishkumar2804** [comment]
doesn't work.

**Comment by DigitalTAC** [comment]
Just to make sure, are you calling it inside
deviceready or
platform.ready (ionic)?

**Comment by ashishkumar2804** [comment]
Yes, it was working fine till cordova ios version 5.x. on upgrading to cordova ios 6.x version. it started breaking. I removed and added the plugin again..but still no luck.
Not sure..what i am missing.
